pkg/apis/velero/v1/pod_volume_backup_types.go

```diff
@@ -134,6 +134,7 @@ type PodVolumeBackupStatus struct {
 // +kubebuilder:printcolumn:name="Started",type="date",JSONPath=".status.startTimestamp",description="Time duration since this PodVolumeBackup was started"
 // +kubebuilder:printcolumn:name="Bytes Done",type="integer",format="int64",JSONPath=".status.progress.bytesDone",description="Completed bytes"
 // +kubebuilder:printcolumn:name="Total Bytes",type="integer",format="int64",JSONPath=".status.progress.totalBytes",description="Total bytes"
+// +kubebuilder:printcolumn:name="Incremental Bytes",type="integer",format="int64",JSONPath=".status.progress.incrementalBytes",description="Incremental bytes"
```

Comment:
See https://github.com/vmware-tanzu/velero/pull/9357/files#r2450263424
I don't suggest we add it here, users only need to know it when the backup completes, so it is enough to show it in kubectl describe

Reply:
@Lyndon-Li although we could add this as a printer column with a lower priority, so it only show up via `kubectl get -o wide`.

Reply:
I've added `priority=10` so you only see it with `kubectl get -o wide`. Regular `kubectl get` won't show it.